ngraph/core/reference/src/runtime/reference/depth_to_space.cpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/runtime/reference/depth_to_space.hpp"
+#include <cmath>
+#include <numeric>
+#include "ngraph/check.hpp"
+#include "ngraph/runtime/opt_kernel/reshape.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            void depth_to_space(const char* in,
+                                const Shape& in_shape,
+                                char* out,
+                                const Shape& out_shape,
+                                const std::size_t block_size,
+                                const op::DepthToSpace::DepthToSpaceMode mode,
+                                const size_t elem_size)
```

Comment:
nitpick: combination of `std::size_t` and `size_t`

Reply:
stick to `size_t`